include/mbedtls/ssl.h

```diff
@@ -84,8 +84,13 @@
 #define MBEDTLS_ERR_SSL_BAD_CERTIFICATE                   -0x7A00
 /* Error space gap */
 /**
```

Comment:
Missing: changelog entry

Reply:
In particular, for people (if any) who had working code handling 1.3 session tickets with 3.6.0, we must explicitly draw attention to the fact that they need to change their code. (Having to change your working code to keep it working is extremely unexpected in an LTS.)